.github/container/test-pax.sh

```diff
@@ -75,14 +78,15 @@ while [ : ]; do
             ENABLE_TE=1
             shift 1
             ;;
+        --enable-fmha)
+            ENABLE_FMHA="$2"
+	    NVTE_FUSED_ATTN=1
```

Comment:
I think this should be `NVTE_FUSED_ATTN="$2"`

Reply:
Updated